### PR TITLE
Fv3gfs cime build

### PR DESCRIPTION
**Pull request by jedwards4b**
CIME_MODEL=ufs now builds the standalone fv3gfs model with nems. 

Test suite: scripts_regression_tests.py (in progress)
Test baseline: 
Test namelist changes: 
Test status: bit for bit

Fixes 

User interface changes?: 

Update gh-pages html (Y/N)?:

Code review:

### CODE REVIEW

**Comment by jedwards4b** [comment]
@billsacks @mvertens I am confused as to the origin of some of the changes on this branch - can you please review and make sure that these updates to master are okay.

File: scripts/Tools/bless_test_results [hunk 1]
```diff
@@ -74,6 +74,12 @@ OR
     parser.add_argument("-r", "--test-root", default=default_testroot,
                         help="Path to test results that are being blessed")
 
+    parser.add_argument("--new-test-root",
```
Comment (jgfouca) [inline scripts/Tools/bless_test_results]: Didn't these changes already go into master in a previous PR?

**Comment by jgfouca** [comment]
@jedwards4b , it looks like there are serious problems in the history of this branch. For example, my commit "Improve scripts_regression_tests test cleanup" appears in this PR as "96f7fab", but our main repo, it is "cbe1cfb6". I think you must have rebased a bunch of master stuff into your branch.

Unfortunately, there's no easy recovery. You'll have to make a new branch and cherry pick all the commits you want to keep.

**Comment by jedwards4b** [comment]
@jgfouca I was afraid of that - okay, i'll work on it.

**Pull request closed without merging**